protocols/v2/noise-sv2/src/initiator.rs

```diff
@@ -1,3 +1,70 @@
+//! # Noise NX Protocol Initiator
```

Comment:
What is NX?

Reply:
For noise protocol, we have a number of different handshake patterns. NX is the one we are using.